src/main/java/com/bandwidth/iris/sdk/IrisClient.java

```diff
@@ -1,186 +0,0 @@
-package com.bandwidth.iris.sdk;
-
-import com.bandwidth.iris.sdk.model.BaseModel;
-import com.bandwidth.iris.sdk.model.BaseResponse;
-import com.bandwidth.iris.sdk.utils.XmlUtils;
-import org.apache.commons.lang3.StringUtils;
-import org.apache.http.Header;
-import org.apache.http.HttpResponse;
-import org.apache.http.auth.AuthScope;
-import org.apache.http.auth.Credentials;
-import org.apache.http.auth.UsernamePasswordCredentials;
-import org.apache.http.client.methods.*;
-import org.apache.http.client.utils.URIBuilder;
-import org.apache.http.entity.ContentType;
-import org.apache.http.entity.FileEntity;
-import org.apache.http.entity.StringEntity;
-import org.apache.http.impl.client.DefaultHttpClient;
-import org.apache.http.util.EntityUtils;
-
-import javax.xml.stream.XMLInputFactory;
-import java.io.File;
-import java.net.URISyntaxException;
-import java.util.HashMap;
-import java.util.Map;
-
-/**
- * Created by sbarstow on 9/22/14.
- */
-public class IrisClient {
-
-    private static String defaultUri = "https://api.inetwork.com/v1.0";
-    private static String defaultVersion = "v1.0";
-    protected DefaultHttpClient httpClient;
-    private String baseUserUrl;
-    private String baseUrl;
-    private String accountId;
-    private String uri;
-    private String userName;
-    private String password;
-    private String version;
-    private XMLInputFactory xmlInputFactory = XMLInputFactory.newInstance();
-
-    public IrisClient(String uri, String accountId,
-            String userName, String password, String version) {
-        this.uri = uri;
-        this.accountId = accountId;
-        this.userName = userName;
-        this.password = password;
-        this.version = version;
-
-        httpClient = new DefaultHttpClient();
```

Comment:
HttpClient is highly configurable (think SSL/TLS...). It might be something that users should be able to pass in. If none is passed in a default one could be created.